utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt

```diff
@@ -78,7 +78,7 @@ class UrlImageParser private constructor(
     }
   }
 
-  private fun calculateInitialMargin(drawableWidth: Int): Int {
+  public fun calculateInitialMargin(drawableWidth: Int): Int {
```

Comment:
We shouldn't make internal methods public for testing purposes. Instead, we should test through the public API.

In this case, could we verify the drawable's bounds in the test to make sure it's calculated correctly?

Reply:
We need to have testing image to get drawable bounds. So I was testing with default values. We can have it tested in #277 .